python/tvm/relay/op/nn/_nn.py

```diff
@@ -620,3 +620,12 @@ def pad_shape_func(attrs, inputs, _):
 reg.register_shape_func("nn.bias_add", False, elemwise_shape_func)
 reg.register_shape_func("nn.softmax", False, elemwise_shape_func)
 reg.register_shape_func("nn.relu", False, elemwise_shape_func)
+
```

Comment:
move the following register above the shape func section.

Reply:
Thanks for suggestion. Updated.